src/profile/v100/address/address_format.rs

```diff
@@ -14,4 +14,5 @@ pub enum AddressFormat {
     Full,
     Raw,
     Default,
+    Hidden,
```

Comment:
Can we try finding a more descriptive name of the variant (Rust calls "cases" "variants") than `Hidden`? I think `Hidden` is a bit ambigous, "format AccountAddress as Hidden" makes it sound like it will be the empty string, or a `"****"` string. So maybe, `Middle` or `GetMiddle` or `ExtractMiddle` or `GetOmitted` or `ExtractOmitted` etc. or "Center" instead of "Middle". Hmm or "Dropped" "Cut" "Ignored" are alternatives to "Omitted". `GetCutBody`? hmm

Reply:
https://www.thesaurus.com/browse/omit (I love using thesaurus)

Reply:
Makes sense to use something different than `Hidden`. I like just `Middle` given the other ones (`Full`, `Raw` & `Default`) are just adjectives (and have no verbs as the other proposals).

Reply:
yeah sure, just `Middle` works.